examples/server/server.cpp

```diff
@@ -1565,16 +1565,16 @@ struct server_queue {
         }
         QUE_DBG("new task, id = %d, front = %d\n", task.id, front);
         if (front) {
-            queue_tasks.push_front(std::move(task));
+            queue_tasks.push_front(task);
         } else {
-            queue_tasks.push_back(std::move(task));
+            queue_tasks.push_back(task);
```

Comment:
Without these moves, the task will be copied here. Better to keep them.

Reply:
Thanks, seems like I mistakenly removed this move. Added them back in https://github.com/ggml-org/llama.cpp/pull/12936/commits/240ea2451cf6986b1268c59e69d6a849701c86c6